change/@fluentui-react-charting-8640e1f3-a73f-4fba-80d5-0298db6d6cd8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "reverted back the required changes",
```

Comment:
Provide a proper comment in the changelog

Reply:
Okay, this comment was picked from my last commit. I'll change this